### PR TITLE
[Kotlin Migration] MultimediaEditFieldActivityTestBase

### PR DESCRIPTION
**Pull request by Akshay0701**

### CODE REVIEW

File: AnkiDroid/src/test/java/com/ichi2/anki/multimediacard/activity/MultimediaEditFieldActivityTestBase.kt [hunk 1]
```diff
@@ -13,72 +13,65 @@
...
     }
 
-    protected MultimediaEditFieldActivity setupActivityMock(IFieldController controller, MultimediaEditFieldActivity editFieldActivity) {
-        MultimediaEditFieldActivity activity = Mockito.mock(MultimediaEditFieldActivity.class);
-        when(activity.getResources()).thenReturn(editFieldActivity.getResources());
-        controller.setEditingActivity(activity);
-        return activity;
+    protected fun setupActivityMock(controller: IFieldController, editFieldActivity: MultimediaEditFieldActivity): MultimediaEditFieldActivity {
+        val activity = Mockito.mock(MultimediaEditFieldActivity::class.java)
+        Mockito.`when`(activity.resources).thenReturn(editFieldActivity.resources)
```
Comment (david-allison-1) [inline AnkiDroid/src/test/java/com/ichi2/anki/multimediacard/activity/MultimediaEditFieldActivityTestBase.kt:73]: We typically want to use the `mockito-kotlin` library to remove the `when`. 

Could you `@KotlinCleanup` here (maybe we need a disabled lint check for this as it's a common issue/operation - your call).

**Review by david-allison-1 — CHANGES_REQUESTED** [review]
On tests: if they pass and comments aren't removed, I'm happy

File: AnkiDroid/src/test/java/com/ichi2/anki/multimediacard/activity/MultimediaEditFieldActivityTestBase.kt (hunk 1 above)
Comment (Akshay0701) [reply AnkiDroid/src/test/java/com/ichi2/anki/multimediacard/activity/MultimediaEditFieldActivityTestBase.kt:73]: do i need to use `[this](https://github.com/mockito/mockito-kotlin)` libary

Comment (mikehardy) [reply AnkiDroid/src/test/java/com/ichi2/anki/multimediacard/activity/MultimediaEditFieldActivityTestBase.kt]: I'm not sure this is actually a valid comment, the scoped storage area uses when also, and I haven't seen any references to `mockito-kotlin` in the codebase when I do full text search

no results on mockito-kotlin
```
mike@bistromath:~/work/AnkiDroid/Anki-Android-Upstream/AnkiDroid/src (master) % grep -ri mockito-kotlin *
mike@bistromath:~/work/AnkiDroid/Anki-Android-Upstream/AnkiDroid/src (master) % 
```

lots of results on when in kotlin files
```
test/java/com/ichi2/anki/multimediacard/fields/BasicImageFieldControllerTest.kt:import org.mockito.Mockito.`when`
test/java/com/ichi2/anki/servicelayer/scopedstorage/MoveFileTest.kt:            Mockito.doAnswer { Timber.w("testing: do nothing on copy") }.`when`(it).copyFile(any(), any())
test/java/com/ichi2/anki/servicelayer/scopedstorage/MoveFileTest.kt:                Mockito.doThrow(TestException("test-copyFile()")).`when`(it).copyFile(any(), any())
test/java/com/ichi2/anki/servicelayer/scopedstorage/MoveFileTest.kt:            Mockito.doThrow(TestException("test-deleteFile()")).`when`(it).deleteFile(any())
test/java/com/ichi2/anki/servicelayer/scopedstorage/MoveFileTest.kt:                Mockito.doThrow(TestException("test-deleteFile()")).`when`(it).deleteFile(any())
```

Comment (mikehardy) [reply AnkiDroid/src/test/java/com/ichi2/anki/multimediacard/activity/MultimediaEditFieldActivityTestBase.kt]: kotlin cleanup was added

**Review by mikehardy — APPROVED** [review]
LGTM - pending local check after rebasing to current master

**Review by david-allison — APPROVED** [review]
If CI passes, I'm happy

**Comment by mikehardy** [comment]
went through locally no problem vs API24 / API31, CI e2e was a flake, going to merge

**Pull request merged**